### DISCUSSION TITLE
Idea/Question: Postfix keyword that makes the command not saved in the history

### DISCUSSION THREAD

**Discussion by sigmaSd** [fish-shell/fish-shell#10098] [General]
I like to use the space trick to not save some commands `<space> command here wont be saved`, but I feel this can be made more convenient by allowing a postfix keyword for the same purpose . In practice I keep writing the command while forgetting  space so I have to go to the start and add it.

I thought a keyword would be an easy way to add this: `git reset --hard @nosave` or a shortcut `git reset --hard @ns` (less typing is better?)

**Comment by faho** [comment]
This would be the only postfix keyword in the entire language.

So I don't believe we're going to do this.
